<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPdfClass</class>
 <widget class="QMainWindow" name="ExportPdfClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExportPdf</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>431</width>
      <height>421</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>qrc:/ExportPdf/test.html</string>
     </url>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtTop</enum>
    </property>
    <item>
     <property name="text">
      <string>qqq</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>www</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>aaa</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_toPdf"/>
    <addaction name="action_toWord"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_toPdf">
   <property name="text">
    <string>导出pdf</string>
   </property>
  </action>
  <action name="action_toWord">
   <property name="text">
    <string>导出word</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="exportpdf.qrc"/>
 </resources>
 <connections/>
</ui>
